pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_improving-search-results-with-abstract.adoc

```diff
@@ -0,0 +1,15 @@
+:_module-type: CONCEPT
+
+[id="con_improving-search-results-with-abstract_{context}"]
+= Role of abstract in search engine results
+
+[role="_abstract"]
+In {ProductShortName}, each source file includes an abstract that provides search engines with metadata about your content. The abstract describes the purpose of document and also helps the reader to determine whether the content is useful to them.
+
+{ProductShortName} automatically detects the first paragraph of a module or an assembly as the abstract. However, you can add the `[role="_abstract"]` tag in the source files to specify the paragraph that you want to use as an abstract. Ensure that the abstract immediately follows the tag, with no blank lines in between. The `[role="_abstract"]` tag is optional.
```

Comment:
Good questions @stoobie I think you're right. I believe the abstract tag lets you choose the paragraph that you want to use as the abstract. @redhataccess/eud-tech-review could you please confirm this?

Reply:
@stoobie @dvagnero 
Yes, in case you want to mark a paragraph other than the first one as the abstract, you can mark it with the `[role="_abstract"]` tag